input/pagecontent/StructureDefinition-mcode-brachytherapy-treatment-phase-intro.md

```diff
@@ -18,7 +34,7 @@ The following table shows valid combinations of modality and technique for brach
 **Key:**
 
 * IORT: Intraoperative radiation therapy (SCT#168524008 Radiotherapy - intraoperative control)
-* IMB: Intensity Modulated Brachytherapy
+* IMB: Intensity Modulated Intercavitary Brachytherapy
```

Comment:
Why is IMB limited to Intercavitary? It seems like IMB could be performed on interstitial or intravascular tissue as well.  Are we missing a term?

Reply:
MVS sent this out for review...hopefully we'll get feedback from that.   intercavitory is a typo --> intra, not inter.   That is the definition.

Reply:
Changed to "Intracavitary".